src/main/flight/pid.c

```diff
@@ -1294,6 +1299,7 @@ void FAST_CODE pidController(const pidProfile_t *pidProfile, timeUs_t currentTim
 #if defined(USE_ACC)
     const bool gpsRescueIsActive = FLIGHT_MODE(GPS_RESCUE_MODE);
     const bool levelModeActive = FLIGHT_MODE(ANGLE_MODE) || FLIGHT_MODE(HORIZON_MODE) || gpsRescueIsActive;
+    const bool levelRaceModeActive = levelRaceMode && (FLIGHT_MODE(ANGLE_MODE) || FLIGHT_MODE(HORIZON_MODE));
```

Comment:
I still don't think this will disable when GPS Rescue is active. The `FLIGHT_MODE(ANGLE_MODE)` (or `HORIZON_MODE`) can still be active when GPS Rescue is (user might have been in angle mode before engaging GPS Rescue for example). Looks like you need ` && !gpsRescueIsActive`.

Reply:
I assumed these flight modes are mutually exclusive, fixed.